src/main/java/com/microsoft/azure/functions/worker/WorkerLogManager.java

```diff
@@ -28,20 +29,33 @@ private void initializeImpl(JavaWorkerClient client, boolean logToConsole) {
         assert this.client == null && client != null;
         this.client = client;
         this.logToConsole = logToConsole;
-        addHostClientHandlers(this.hostLogger, null);
+        addHandlers(this.hostLogger, null);
     }
 
-    private void deinitializempl() {
+    private void deinitializImpl() {
         assert this.client != null;
         clearHandlers(this.hostLogger);
         this.logToConsole = false;
         this.client = null;
     }
 
+    private void flushLogsImpl() {
+        flush(emptyLogger);
```

Comment:
we may can remove flush this emptyLogger as it was not used anywhere

Reply:
Yes, this can be removed